### ISSUE TITLE
Can't see the Writing Data Statistics

### ISSUE THREAD

**Issue by fagnercarvalho** [apache/iotdb#2201]
Can't see the Writing Data Statistics as timeseries even after setting `enable_stat_monitor` as `true` in `iotdb-engine.properties`

1. Create the following Dockerfile (make sure to set `enable_stat_monitor` as `true` in `iotdb-engine.properties`):
```Docker
FROM apache/iotdb:0.11.0

COPY iotdb-engine.properties /iotdb/conf/

ENTRYPOINT ["/iotdb/sbin/start-server.sh"]
```
2. Build the `Dockerfile` and run it.
```
docker build -t iotdb .
docker run -d -p 0.0.0.0:6667:6667 -p 0.0.0.0:31999:31999 -p 0.0.0.0:8181:8181 --name custom-iotdb iotdb
```
3. Go to the IoTDB CLI inside the container.
```
docker exec -it custom-iotdb /bin/bash
start-cli.sh
```
4. Run a few commands to populate two timeseries with data:
```
set storage group to root.ln

create timeseries root.ln.wf02.wt02.status with datatype=BOOLEAN,encoding=RLE
create timeseries root.ln.wf02.wt02.temperature with datatype=FLOAT,encoding=RLE

insert into root.ln.wf02.wt02(time,temperature,status) values(2017-11-01T16:37:50.000,30.34,true)
insert into root.ln.wf02.wt02(time,temperature,status) values(2017-11-01T17:00:50.000,20.34,false)
```
5. Run `SHOW TIMESERIES` 

**Expected behavior**
I was expecting to see the `root.stats.write.global.TOTAL_POINTS` and `root.stats.write.root_ln.TOTAL_POINTS` timeseries (as indicated in the [Monitor and Log Tools](https://iotdb.apache.org/UserGuide/V0.11.x/System%20Tools/Monitor%20and%20Log%20Tools.html) page) but the only new timeseries I see are `root.stats.file_size.WAL` and `root.stats.file_size.SYS`.

 - OS: Windows but using Docker
 - Version 0.11.0

**Comment by HTHou** [comment]
Actually this module is unavailable in 0.11 I think. A PR https://github.com/apache/iotdb/pull/1725 has fix it and should be released with 0.12.  If you would like to use it, you can try to build the master branch. :)

**Comment by fagnercarvalho** [comment]
I tried apache/iotdb:latest that was pushed to https://hub.docker.com/r/apache/iotdb 12 days ago and PR #1725 was pushed 13 days ago. I guess that this change should be there but I'm still not seeing in the time series.

**Comment by HTHou** [comment]
Eh, I don't think the apache/iotdb:latest on https://hub.docker.com/r/apache/iotdb contains the PR #1725, since this PR is only merged into master branch and the 0.12 is not released. The docker image of apache/iotdb:latest should still be 0.11.0...

**Comment by fagnercarvalho** [comment]
Ah, got it! I will try the version in master later today and I will let you know if these are available there.

**Comment by Alima777** [comment]
Hi, do not forget to see the latest doc on http://iotdb.apache.org/UserGuide/Master/System%20Tools/Monitor%20and%20Log%20Tools.html.
Some modules before are unavailable now.

**Comment by fagnercarvalho** [comment]
Hey everyone!


Unfortunately I didn't have success to show the `TOTAL_POINTS` stats.

First I tried to run a Docker container from master but got this error message:

```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/iotdb/sbin/start-server.sh\": stat /iotdb/sbin/start-server.sh: no such file or directory": unknown.
```

Here are the commands I executed:
```
docker build -t iotdb .
docker run -d -p 0.0.0.0:6667:6667 -p 0.0.0.0:31999:31999 --name custom-iotdb-2 iotdb
```

Here is the Dockerfile:
```
FROM openjdk:11-jre-slim
RUN apt update \
  # procps is for `free` command
  && apt install wget unzip lsof procps -y \
  && wget https://github.com/apache/iotdb/archive/master.zip \
  #&& wget https://downloads.apache.org/iotdb/0.11.1/apache-iotdb-0.11.1-bin.zip \
  # if you are in China, use the following URL
  #&& wget https://mirrors.tuna.tsinghua.edu.cn/apache/iotdb/0.11.0/apache-iotdb-0.11.0-bin.zip \
  && unzip master.zip \
  && rm master.zip \
  && mv iotdb-master /iotdb \
  && apt remove wget unzip -y \
  && apt autoremove -y \
  && apt purge --auto-remove -y \
  && apt clean -y
EXPOSE 6667
EXPOSE 31999
EXPOSE 5555
EXPOSE 8181
VOLUME /iotdb/data
VOLUME /iotdb/logs
ENV PATH="/iotdb/sbin/:/iotdb/tools/:${PATH}"

COPY iotdb-engine.properties /iotdb/conf/

#ENTRYPOINT ["/iotdb/sbin/start-server.sh"]
```

I also tried using version 0.11.1 released recently. This time I was able to run the CLI and run commands but I still can't see any stats besides `WAL` and `SYS`.

Dockerfile:
```
FROM openjdk:11-jre-slim
RUN apt update \
  # procps is for `free` command
  && apt install wget unzip lsof procps -y \
  && wget https://downloads.apache.org/iotdb/0.11.1/apache-iotdb-0.11.1-bin.zip \
  # if you are in China, use the following URL
  #&& wget https://mirrors.tuna.tsinghua.edu.cn/apache/iotdb/0.11.0/apache-iotdb-0.11.0-bin.zip \
  && unzip apache-iotdb-0.11.1-bin.zip \
  && rm apache-iotdb-0.11.1-bin.zip \
  && mv apache-iotdb-0.11.1 /iotdb \
  && apt remove wget unzip -y \
  && apt autoremove -y \
  && apt purge --auto-remove -y \
  && apt clean -y
EXPOSE 6667
EXPOSE 31999
EXPOSE 5555
EXPOSE 8181
VOLUME /iotdb/data
VOLUME /iotdb/logs
ENV PATH="/iotdb/sbin/:/iotdb/tools/:${PATH}"

COPY iotdb-engine.properties /iotdb/conf/

ENTRYPOINT ["/iotdb/sbin/start-server.sh"]
```

In the iotdb-properties.conf I set:
```
enable_stat_monitor=true
enable_monitor_series_write=true
enable_performance_stat=true
enable_performance_tracing=true
```

Here is the link for the complete iotdb-properties.conf: https://pastebin.com/CT7PUNpE.

I tried running all the commands below but no stats besides `WAL` and `SYS` are available in the timeseries.

```
set storage group to root.ln

create timeseries root.ln.wf02.wt02.status with datatype=BOOLEAN,encoding=RLE
create timeseries root.ln.wf02.wt02.temperature with datatype=FLOAT,encoding=RLE

insert into root.ln.wf02.wt02(time,temperature,status) values(2017-11-01T16:37:50.000,30.34,true)
insert into root.ln.wf02.wt02(time,temperature,status) values(2017-11-01T17:00:50.000,20.34,false)

SELECT * FROM root.ln.wf02

SELECT COUNT(*) FROM root.ln.wf02

SHOW TIMESERIES
```

What I am doing wrong?

**Comment by HTHou** [comment]
Hi, actually the version 0.11.1 you tried still does not support the monitor feature.... The next 0.11.x will be the bug-fixed version of 0.11.1, we are not going to put new features in that version.

If you'd like to try, you can try to build the master branch as I said before. And I can tell that the 0.12 is coming soon, maybe next month. 😀
